src/content/static/components/CookiesData/CookiesTable.tsx

```diff
@@ -7,19 +7,18 @@ const CookiesTable = ({ headers, data, caption }: CookiesDataProps) => (
     <thead>
       <tr>
         {headers.map((header: string) => (
-          <th>
-            <ic-typography variant="subtitle-small">{header}</ic-typography>
-          </th>
+          <th key={header}>
+          <ic-typography variant="subtitle-small">{header}</ic-typography>
+        </th>
+      ))}
+    </tr>
+  </thead>
+  <tbody>
+    {data.map((cookieData: any) => (
+      <tr key={cookieData.id}>
```

Comment:
The `id` property doesn't exist on `cookieData` so this will never be set. Should it be updated to `key`?

Reply:
Also, this file may need prettier ran on it. The indentation doesn't line up. Don't know how the CI didn't catch it, it fails for me when there aren't even any changes.